backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/nodes/AbstractPackageNodeDescriptionProvider.java

```diff
@@ -204,6 +205,10 @@ private DropNodeTool createDropFromDiagramTool(IViewDiagramElementFinder cache)
     }
 
     private NodeTool createNodeTool(NodeDescription nodeDescription, EClass eClass) {
+        if (SysmlPackage.eINSTANCE.getNamespaceImport().equals(eClass)) {
```

Comment:
At first sight it seems strange to have this code. It looks like a "patch".
Please check twice if there is another better solution.

Reply:
Changed with `addCustomNodes` mechanism